client/packages/system/src/Item/api/api.ts

```diff
@@ -62,7 +62,11 @@ export const getItemQueries = (sdk: Sdk, storeId: string) => ({
         key: itemParsers.toSortField(sortBy),
         isDesc: sortBy.isDesc,
         storeId,
-        filter: { ...filterBy, type: { equalTo: ItemNodeType.Stock } },
+        filter: {
+          ...filterBy,
+          type: { equalTo: ItemNodeType.Stock },
+          isVisible: true,
```

Comment:
I found some items had appeared in my database ( due to sync testing ) which weren't valid, and had `isVisible` false but still showed. Seems reasonable to only show visible items?